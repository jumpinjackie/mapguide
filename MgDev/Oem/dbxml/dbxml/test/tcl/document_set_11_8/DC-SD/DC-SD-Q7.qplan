<XQuery>
  <FLWOR>
    <ForBinding name="item">
      <QueryPlanToAST>
        <ChildJoinQP>
          <StepQP axis="child" name="catalog" nodeType="element">
            <ValueQP index="node-element-equality-string" operation="eq" child="name_of_country" value="Canada"/>
          </StepQP>
          <StepQP axis="parent-of-child" name="item" nodeType="element">
            <ExceptQP>
              <StepQP axis="parent-of-child" name="authors" nodeType="element">
                <StepQP axis="parent-of-child" name="author" nodeType="element">
                  <StepQP axis="parent-of-child" name="contact_information" nodeType="element">
                    <StepQP axis="parent-of-child" name="mailing_address" nodeType="element">
                      <ValueFilterQP comparison="eq" general="true">
                        <SequentialScanQP nodeType="element" name="name_of_country"/>
                        <Sequence>
                          <AnyAtomicTypeConstructor value="Canada" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
                        </Sequence>
                      </ValueFilterQP>
                    </StepQP>
                  </StepQP>
                </StepQP>
              </StepQP>
              <StepQP axis="parent-of-child" name="authors" nodeType="element">
                <StepQP axis="parent-of-child" name="author" nodeType="element">
                  <StepQP axis="parent-of-child" name="contact_information" nodeType="element">
                    <ExceptQP>
                      <SequentialScanQP nodeType="element" name="mailing_address"/>
                      <StepQP axis="parent-of-child" name="mailing_address" nodeType="element">
                        <ValueFilterQP comparison="eq" general="true">
                          <SequentialScanQP nodeType="element" name="name_of_country"/>
                          <Sequence>
                            <AnyAtomicTypeConstructor value="Canada" typeuri="http://www.w3.org/2001/XMLSchema" typename="string"/>
                          </Sequence>
                        </ValueFilterQP>
                      </StepQP>
                    </ExceptQP>
                  </StepQP>
                </StepQP>
              </StepQP>
            </ExceptQP>
          </StepQP>
        </ChildJoinQP>
      </QueryPlanToAST>
    </ForBinding>
    <QueryPlanToAST>
      <VariableQP name=":item"/>
    </QueryPlanToAST>
  </FLWOR>
</XQuery>
