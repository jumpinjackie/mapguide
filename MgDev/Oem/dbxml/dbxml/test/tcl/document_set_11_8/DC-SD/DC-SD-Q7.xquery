for $item in collection("11.8.2.dbxml")/catalog/item
where every $add in
    $item/authors/author/contact_information/mailing_address
satisfies $add/name_of_country = "Canada"
return
    $item
