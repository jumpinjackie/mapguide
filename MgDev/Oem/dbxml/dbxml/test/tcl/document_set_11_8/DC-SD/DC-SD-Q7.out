<item id="I2"><title>BABABABARIALREDogsshallseem.</title>
<authors><author><name><first_name>Shinsuke</first_name>
<middle_name>sly platelets mu</middle_name>
<last_name>BABABABABANGUL</last_name>
</name>
<date_of_birth>1825-04-02</date_of_birth>
<biography>stealthy, dogged sheaves shall lose slow dinos:careful, careful patterns past the grouches would hang without the idle forges;orbit</biography>
<contact_information><mailing_address><street_information><street_address>/f1b.{z.,.7,mwKtY4qf27c)#!+I)I</street_address>
</street_information>
<name_of_city>Cody</name_of_city>
<name_of_state>District Of Columbia</name_of_state>
<zip_code>94064</zip_code>

<name_of_country>Canada</name_of_country>
</mailing_address>
<phone_number>+91(821)26632638</phone_number>
<email_address>BABABABABANGUL@filemaker.com</email_address>
</contact_information>
</author>
<author><name><first_name>Marina</first_name>
<middle_name>dino</middle_name>
<last_name>BABABABABARERE</last_name>
</name>
<date_of_birth>1814-06-26</date_of_birth>
<biography>frets outside the finally close waters will run ruthlessly slow dugouts--quick players inside the courts eat ruthlessly with the bold, dogged sentiments;orbits promise--careful tithes up the dogged foxes hinder ironically ruthlessly fluffy waters--even theodolites outside the ironic, enticing escapades maintain a</biography>
<contact_information><mailing_address><street_information><street_address>Z; ,UK6YDow8nBK?3iYd#Lo: u+[b$nCh-]MS</street_address>
<street_address>EnbY2+kvs8 ehQgfL9WQEV@R@</street_address>
</street_information>
<name_of_city>Columbia</name_of_city>
<name_of_state>Delaware</name_of_state>
<zip_code>G1C 9C8</zip_code>

<name_of_country>Canada</name_of_country>
</mailing_address>
<phone_number>+78(321)94777024</phone_number>
<email_address>BABABABABARERE@co.in</email_address>
</contact_information>
</author>
<author><name><first_name>Hansong</first_name>
<middle_name>enticing, ca</middle_name>
<last_name>BABABABABAULNGgfL</last_name>
</name>
<date_of_birth>1822-01-28</date_of_birth>
<biography>slyly furious ideas thrash closely orbits!never fluffy warhorses are permanent, fluffy forges:slow, stealthy gifts could have to grow ironically across the brave grouches--furious, regular pearls past the regular, enticing decoys engage bold asymptotes.idly fluffy notornis nag bravely.ironic sheaves against the quiet orbits nag thin courts:dependencie</biography>
<contact_information><mailing_address><street_information><street_address>8~?3LFNt~_EMG-52E)icKnk!in+</street_address>
</street_information>
<name_of_city>Lorient</name_of_city>
<name_of_state>Vermont</name_of_state>
<zip_code>84434</zip_code>

<name_of_country>Canada</name_of_country>
</mailing_address>
<phone_number>+28(673)86834120</phone_number>
<email_address>BABABABABAULNGgfL@filelmaker.com</email_address>
</contact_information>
</author>
<author><name><first_name>Esther</first_name>
<middle_name>bold Tiresia</middle_name>
<last_name>BABABABAOGRING</last_name>
</name>
<date_of_birth>1898-09-24</date_of_birth>
<biography>busy, permanent excuses belive ironically without the brave Tiresias.daring, close dinos alongside of the slowly sly players affix busily into the blithe players?quick, quick Tiresias at the finally silent players would hang boldly depths?players against the somas shall have to snooze under the ironic, furious waters.silent, brave multipliers can mold boldly?furious, daring players against the stealthly dogged epitaphs nag blithely for the ironic platelets.dependencies detect</biography>
<contact_information><mailing_address><street_information><street_address>l:VBNz[avYU(M:GX?s#3qUti</street_address>
</street_information>
<name_of_city>Amarillo</name_of_city>
<name_of_state>West Virginia</name_of_state>
<zip_code>41823</zip_code>

<name_of_country>Canada</name_of_country>
</mailing_address>
<phone_number>+64(620)24427834</phone_number>
<email_address>BABABABAOGRING@ac.kr</email_address>
</contact_information>
</author>
</authors>
<date_of_release>1932-07-07</date_of_release>
<publisher><name>enticing, quick dino</name>
<contact_information><mailing_address><street_information><street_address>=q0Mgg#p+{dFi,FARqTSY</street_address>
<street_address>SVNTA554mGy(5z+xS=Y</street_address>
</street_information>
<name_of_city>Norfolk</name_of_city>
<name_of_state>California</name_of_state>
<zip_code>99641</zip_code>

<country><name>Iceland</name>
<exchange_rate>74.147</exchange_rate>
<currency>Krona</currency>
</country>
</mailing_address>
<FAX_number>+10(64)65087848</FAX_number>
<phone_number>+68(943)80939416</phone_number>
<web_site>http://www.ac.kr</web_site>
</contact_information>
</publisher>
<subject>SPORTS</subject>
<description>close escapades mold finally hockey--gifts for the permanent hockey must unwind finally to the blithe escapades:slowly careful attainments beside the regularly busy hockey solve regularly across the slow dugouts:idly slow foxes thrash:sly escapades sublate always up the quick, quick forges?furious, final theodolites do doubt Tiresias.daring instruc</description>
<related_items><related_item><item_id>I648</item_id>
</related_item>
</related_items>
<media><thumbnail><data/>
</thumbnail>
<image><data/>
</image>
</media>
<pricing><suggested_retail_price currency="Dollars">1808.80</suggested_retail_price>
<cost currency="Dollars">999.90</cost>
<when_is_available>1932-07-28</when_is_available>
<quantity_in_stock>13</quantity_in_stock>
</pricing>
<attributes><ISBN>3580444236195</ISBN>
<number_of_pages>9161</number_of_pages>
<type_of_book>PAPERBACK</type_of_book>
<size_of_book><length unit="Inch">56.1</length>
<width unit="Inch">96.7</width>
<height unit="Inch">42.8</height>
</size_of_book>
</attributes>
</item>